### PR TITLE
Added Desktop app MSI installer tip

### PR DESCRIPTION
**Pull request by cwarnermm**

### CODE REVIEW

**Comment by cwarnermm** [comment]
@devinbinnie - 2 questions:
- our [MSI install docs](https://docs.mattermost.com/install/desktop-msi-installer-and-group-policy-install.html) for the desktop app specifically reference the [version v5.3.1](https://github.com/mattermost/desktop/releases/tag/v5.3.1) release. Can admins use other versions of the desktop app?
- is this functionality still in beta?

**Comment by devinbinnie** [comment]
> * our [MSI install docs](https://docs.mattermost.com/install/desktop-msi-installer-and-group-policy-install.html) for the desktop app specifically reference the [version v5.3.1](https://github.com/mattermost/desktop/releases/tag/v5.3.1) release. Can admins use other versions of the desktop app?

Yes, we've had an MSI for a while actually now, but v5.3.1 is currently the latest (at time of writing)

> * is this functionality still in beta?

Technically yes, we were going to have some additional work on it before claiming full support, but it's not currently scheduled.

File: source/install/desktop-msi-installer-and-group-policy-install.rst [hunk 1]
```diff
@@ -120,11 +120,15 @@ Install the Mattermost Desktop App using the MSI installer
...
 
-3. Select ``Finish`` when the installation is complete.
+3. Select **Finish** when the installation is complete.
 
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00017.png
       :alt: When the installation is complete, select Finish.
 
+.. tip::
+
+   To prevent Mattermost from adding a desktop shortcut for the desktop app, open the MSI file with an editor, delete the desktop shortcut code, and save that change as an MST file. Install the MSI using the MST file.
```
Comment (devinbinnie) [inline source/install/desktop-msi-installer-and-group-policy-install.rst:130]: I'd never heard of this process. Where did it come from?

Comment (cwarnermm) [reply source/install/desktop-msi-installer-and-group-policy-install.rst:130]: @devinbinnie - It was from a [peer-to-peer discussion](https://community.mattermost.com/core/pl/bdchci7qd3dxmktonm74n8qika) on the Mattermost Community Server. Please let me know if this approach is something Mattermost would officially support. If it's not, happy to remove.

Comment (devinbinnie) [reply source/install/desktop-msi-installer-and-group-policy-install.rst:130]: Gotcha, so this isn't an officially supported use case by us, but I guess it does happen to work. 0/5 on whether we keep it or not.

**Review by devinbinnie — APPROVED** [review]

**Pull request merged**